pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -335,6 +347,204 @@ def parse_unity_menu_from_str(s: str) -> List[UnittestMenuCase]:
 
         return test_menu
 
+    @property
+    def test_menu(self) -> t.List[UnittestMenuCase]:
+        if self._test_menu is None:
+            self._test_menu = self._parse_test_menu()
+            self.serial.hard_reset()
```

Comment:
Do you think it's necessary to log that the test menu is parsed successfully before restart? In case that the restart is caused by other reasons.